src/modules/chat/index.js

```diff
@@ -29,10 +29,14 @@ const EMOTE_MODIFIERS = {
   'v!': 'bttv-emote-modifier-flip-vertical',
   'z!': 'bttv-emote-modifier-zero-space',
   'c!': 'bttv-emote-modifier-cursed',
+  'l!': 'bttv-emote-modifier-rotate-left',
+  'r!': 'bttv-emote-modifier-rotate-right',
   ffzW: 'bttv-emote-modifier-wide',
   ffzX: 'bttv-emote-modifier-flip-horizontal',
   ffzY: 'bttv-emote-modifier-flip-vertical',
   ffzCursed: 'bttv-emote-modifier-cursed',
+  ffzL: 'bttv-emote-modifier-rotate-left',
```

Comment:
Does frankerfacez support these modifiers? If not we should exclude `ffzL` and `ffzR`.

Reply:
done